### DISCUSSION TITLE
Extend cookie consent settings

### DISCUSSION THREAD

**Discussion by martinbira** [squidfunk/mkdocs-material#4557] [Ideas]
To further enhance the cookie consent form and allow the visitor to view all available cookies directly, while not requiring the site author to perform an override of the `consent.html`-partial, I would like to propose an additional setting that would allow for this.

Suggested addition to settings in mkdocs.yml:
`show_cookies`

Suggested change in `consent.html`:

```
<input type="checkbox" class="md-toggle" id="__settings"{% if config.extra.analytics.show_cookies %}checked{% endif %} />
```

**Comment by squidfunk** [comment]
Thanks for suggesting. However, it makes the "Manage settings" button obsolete, because why would a visitor want to toggle the settings if they are visible anyway? We could maybe default to "always show them, when no button to toggle the settings is included".

**Reply by martinbira** [reply]
Yeah, I thought about that too and that sounds like a good solution!

**Reply by squidfunk** [reply]
Added in 006f520f3.
